languages/csharp/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+In C#, data structures that can hold zero or more elements are known as _collections_. An array is a collection with a fixed size and which elements must all be of the same type. Elements in an array can be assigned to and retrieved by index, with the first element's index being zero:
```

Comment:
- Array as bold
- "whose" instead of "which" (we normally personify things in this situation for reasons unknown)
- Reordered the final line to make it parse a little easier.

```suggestion
In C#, data structures that can hold zero or more elements are known as _collections_. An **array** is a collection that has a fixed size and whose contains must all be of the same type. Elements can be assigned to an array or retrieved from it using an index, with the first element's index being zero:
```